presentation/src/main/java/com/yapp/presentation/util/intent/Intent.kt

```diff
@@ -0,0 +1,20 @@
+package com.yapp.presentation.util.intent
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import androidx.core.content.ContextCompat
+import com.yapp.presentation.BuildConfig
+
+fun intentToPlayStore(context: Context) {
```

Comment:
사람마다 코드 스타일이 다르니 맞다 아니다를 논할 수 있지만,, 코틀린의 장점인 확장함수와 범위 지정 함수를 활용하는건 어떨까용? 지금도 충분히 가독성 좋고 직관적인 코드라고 생각해서 제안성으로만 리뷰 드립니다!
```suggestion
fun Context.intentToPlayStore() {
    val packageName = packageName.takeIf { BuildConfig.DEBUG.not() } ?: packageName.replace(".debug", "")
    Intent(Intent.ACTION_VIEW).apply {
        data = Uri.parse("market://details?id=$packageName")
    }.also { intent ->
        ContextCompat.startActivity(this, intent, null)
    }
}
```

Reply:
오 이런 피드백 너무 좋아요.. 코틀린스럽게 코드 작성하기 알려주셔서 감사합니다 ㅎㅎ